src/sidebar/components/Annotation/AnnotationShareControl.tsx

```diff
@@ -127,70 +135,70 @@ function AnnotationShareControl({
   return (
     <div className="relative" ref={shareRef}>
       <IconButton
-        icon="share"
+        icon={ShareIcon}
         title="Share"
         onClick={toggleSharePanel}
         expanded={isOpen}
       />
       {isOpen && (
-        <Card
-          classes={classnames(
-            // Prefer width 96 (24rem) but ensure that component isn't wider
-            // than 85vw
-            'w-96 max-w-[85vw]',
-            // Position this Card above its IconButton. Account for larger
-            // IconButtons in touch interfaces
-            'absolute bottom-8 right-1 touch:bottom-touch-minimum',
-            'space-y-2 p-2',
-            // Cards do not have a border in the clean theme. Turn it back on.
-            'theme-clean:border theme-clean:border-solid theme-clean:border-grey-3'
-          )}
+        <div
```

Comment:
The changes here add an additional container element to apply positioning styles to versus applying all that to the `Card` component.